docs/content/blog/v0.10.fa.md

```
@@ -0,0 +1,247 @@
+- **رندر پیوند**: بهبود مدیریت URL پایه برای ساختارهای پیچیده سایت
+
+---
+
+## راهنمای مهاجرت
+
+- [**نیازمندی‌های نسخه Hugo**](#hugo-version-requirements): نیاز به Hugo v0.146.0+ (نسخه extended) دارد
+- [**تغییرات پیشوند کلاس‌های CSS**](#css-class-prefix-changes): کلاس‌های CSS کامپوننت اکنون از پیشوند سازگار `hextra-` استفاده می‌کنند
+- [**مدیریت دارایی‌ها**](#asset-management-for-katex-and-mermaid): دارایی‌های KaTeX و Mermaid اکنون در زمان ساخت دانلود می‌شوند
+- [**Tailwind CSS v4**](#tailwind-css-v4): کامپایل CSS داخلی اکنون از Tailwind CSS v4.x با پیشوند `hx:` استفاده می‌کند
+
+#### نیازمندی‌های نسخه Hugo
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)